src/pages/RFOX/components/Claim/ClaimSelect.tsx

```diff
@@ -131,31 +171,111 @@ export const ClaimSelect: FC<ClaimSelectProps & ClaimRouteProps> = ({
   setConfirmedQuote,
   setStepIndex,
 }) => {
-  const hasClaims = false // Just a placeholder for now
+  const stakingAssetId = foxOnArbitrumOneAssetId
+  const stakingAsset = useAppSelector(state => selectAssetById(state, stakingAssetId))
+
+  // TODO(gomes): make this programmatic when we implement multi-account
+  const stakingAssetAccountId = useAppSelector(state =>
+    selectFirstAccountIdByChainId(state, stakingAsset?.chainId ?? ''),
+  )
+
+  const stakingAssetAccountAddress = useMemo(
+    () => (stakingAssetAccountId ? fromAccountId(stakingAssetAccountId).account : undefined),
+    [stakingAssetAccountId],
+  )
+
+  const {
+    data: unstakingRequestCountResponse,
+    isSuccess: isUnstakingRequestCountSuccess,
+    isLoading: isUnstakingRequestCountLoading,
+  } = useReadContract({
+    abi: foxStakingV1Abi,
+    address: RFOX_PROXY_CONTRACT_ADDRESS,
+    functionName: 'getUnstakingRequestCount',
+    args: [stakingAssetAccountAddress ? getAddress(stakingAssetAccountAddress) : ('' as Address)], // actually defined, see enabled below
```

Comment:
The solve here would be to wrap everything from here down in another component, and then in the parent, return null if `stakingAssetAccountId` is not defined. This way wwe dont have to useReadContract with an empty address ever

Reply:
I like it. This pattern exists in 3 separate components at present - to keep this PR isolated to fetching claims I'll do this in a follow-up PR.

Reply:
@woodenfurniture I think that's a non-problem we'd be solving, that's what the `enabled` react-query option is for i.e we don't even *run* the query with an empty address ever.
Also: https://github.com/shapeshift/web/issues/6973 which we could use for wagmi as a react-query wrapper as well, see https://github.com/wevm/wagmi/blob/f07b551dcee73c17ceb2cea2025d7bbb2f2097b4/packages/react/package.json#L78